x/wasm/keeper/query_plugins.go

```diff
@@ -189,6 +189,16 @@ func BankQuerier(bankKeeper types.BankViewKeeper) func(ctx sdk.Context, request
 			}
 			return json.Marshal(res)
 		}
+		if request.Supply != nil {
```

Comment:
This looks good. Some happy path test with the mock would be good for regression